src/engine.cc

```diff
@@ -324,6 +328,25 @@ void EngineLoop::RunLoop() {
   UciLoop::RunLoop();
 }
 
+void EngineLoop::Benchmark() {
+  const char* kNodesStr = "Number of nodes for benchmark";
+  options_.Add<IntOption>(kNodesStr, -1, 999999999, "nodes") = 30000;
+  if (!ConfigFile::Init(&options_) || !options_.ProcessAllFlags()) return;
+  try {
+    CmdIsReady();
+    GoParams go_params;
+    go_params.nodes = options_.GetOptionsDict().Get<int>(kNodesStr);
+    auto start = std::chrono::steady_clock::now();
```

Comment:
For benchmarking, I've usually seen std::chrono::high_resolution_clock being used instead.